### PR TITLE
EpiModel Overhaul Updates for 3/22/20

### PR DESCRIPTION
**Pull request by smjenness**
Handling updates in #333

### CODE REVIEW

File: R/net.mod.nwupdate.R [hunk 1]
```diff
@@ -75,7 +75,7 @@ nwupdate.net <- function(dat, at) {
     }
 
     ## Recovery
-    idsRecov <- dat$nw.update$rec$idsRecov
+    idsRecov <- dat$temp$idsRecov
     recovState <- ifelse(dat$control$type == "SIR", "r", "s")
```
Comment (smjenness) [inline R/net.mod.nwupdate.R]: Can we figure out a way to do this without having to write out idsRecov? One solution, is to look up, in the nwupdate fx the setdiff of ids with a recov status on attr versus the network object.

**Comment by smjenness** [comment]
@andsv2 Let's also change the name of the `tgl` control setting to `tergmLite`.

File: R/net.mod.nwupdate.R (hunk 1 above)
Comment (andsv2) [reply R/net.mod.nwupdate.R]: I was going to suggest a time recovered attr like for infection (time infected), but the above works too without needing to add another tracker.

**Comment by smjenness** [comment]
@andsv2 Looks great. Once the query of new recoveries in nwupdate.net (instead of writing to dat$temp) is done, I think this will be ready to merge into _s4.

File: R/net.mod.nwupdate.R [hunk 2]
```diff
@@ -66,9 +66,12 @@ nwupdate.net <- function(dat, at) {
     }
 
     ## Recovery
-    idsRecov <- dat$temp$idsRecov
-    recovState <- ifelse(dat$control$type == "SIR", "r", "s")
     status <- dat$attr$status
+    attr.status <- which(status == "r")
+    nw.status <- get.vertex.attribute(dat$nw, "status")
```
Comment (smjenness) [inline R/net.mod.nwupdate.R]: This would work for SIR models but not SIS models, right?

**Comment by andsv2** [comment]
All Travis checks passing and lingering issues (without Travis flags) addressed. Merging into _s4 branch.

**Pull request merged**